src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -38,32 +43,61 @@
  */
 public class SonarQubeTask extends DefaultTask {
 
-  private Map<String, Object> sonarProperties;
+    private static final Logger LOGGER = Logging.getLogger(SonarQubeTask.class);
+
+    public static final LogOutput LOG_OUTPUT = new LogOutput() {
+        @Override
+        public void log(String formattedMessage, Level level) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "log" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)